---
layout: default
title: research
categories:
 - research

---

<div class="row">
	<div class="col-md-1"></div>
	<div class="col-md-10">
		<h1>Research Overview</h1>
	</div>
	<div class="col-md-1"></div>
</div>


<div class="row">
	<div class="col-md-1"></div>
	<div class="col-md-10">
		<p>
		Our group is interested in understanding the function of microbial communities and how they can be explored for therapeutic purposes. Specifically, we use bioinformatic, genomic and molecular biology tools to discover new therapeutic compounds and identify microbial signatures for early detection of diseases.
		</p>
		<p>
		The human microbiome contributes to the overall health of its host through multiple mechanisms (e.g. nutrient breakdown, colonization resistance to pathogens, etc.). In addition, microbes represent a rich source for the discovery of new therapeutics; for instance, ~70% of antibiotics in use today were obtained from microbes! The overarching goal of our lab is to combine computation and experimental techniques to study microbial communities for the discovery of new therapeutics.
		</p>
		<p>
		Our lab is particularly interested in understanding how microbes interact in their ecosystem, contribute to diseases, and how they suppress the growth of competitors through the secretion of antimicrobials. We aim to use information on the microbiome to predict disease and outcomes, as well as developing ways to modulate it to improve human health.
		</p>
	</div>
	<div class="col-md-1"></div>
</div>


<div class="bigspacer"></div>

<div class="row">
	<div class="col-md-1"></div>
	<div class="col-md-10">
		<h1>Research Projects</h1>
	</div>
	<div class="col-md-1"></div>
</div>

<div class="row">
	<div class="col-md-1"></div>
	<div class="col-md-10">
		<h2>Platform for new antimicrobial discovery</h2>
	</div>
	<div class="col-md-1"></div>
</div>

<div class="row">
	<div class="col-md-1"></div>
	<div class="col-md-10">
		<p >
		Most antibiotics in use today were derived from microbial ecosystems and discovery of new compounds has slowed despite its rich potential: only a small proportion of environmental microbes have been characterized with recent estimates placing this proportion at less than 1%. The difficulty in exploring these microenvironments has been attributed to the huge diversity of unculturable microbes. To circumvent these limitations, we are developing single cell technology for high-throughput identification of antibiotic-producing microbes with no requirement for microbial growth.
		</p>
		
	<div class="col-md-1"></div>
</div>

<div class="bigspacer"></div>

<div class="row">
	<div class="col-md-1"></div>
	<div class="col-md-10">
		<h2>Microbe-specific targeted therapy to modulate the microbiome</h2>
	</div>
	<div class="col-md-1"></div>
</div>

<div class="row">
	<div class="col-md-1"></div>
	<div class="col-md-10">
		<p >
		Dysbiosis in the human microbiome has been associated with several diseases.  In other words, certain microbes have been found to be depleted or enriched in disease conditions. While antibiotics can be used to eliminate ‘detrimental’ microbes, it usually results in depopulation of beneficial commensals. The holy grail of microbiome therapeutics is to specifically target an organism of interest while preserving majority of the microbiome. Using antimicrobial prodrug strategies, we are developing taxa-specific inhibitors of organisms of interest.
		</p>
		
	<div class="col-md-1"></div>
</div>

<div class="bigspacer"></div>

<div class="row">
	<div class="col-md-1"></div>
	<div class="col-md-10">
		<h2>Computational pipelines for functional investigation of the microbiome</h2>
	</div>
	<div class="col-md-1"></div>
</div>

<div class="row">
	<div class="col-md-1"></div>
	<div class="col-md-10">
		<p>
		Studies associating the microbiome with diseases often rely on abundance-based analysis. While this is helpful in identifying differentially enriched microbes between two conditions, it provides little insight into the functional characteristics of the microbiome. We are interested in developing computational tools to study the function of the microbiome such as microbial growth dynamics, global gene expression, and pathway kinetics.
		</p>
		
	</div>
	
	<div class="col-md-1"></div>
</div>
<div class="bigspacer"></div>