---
layout: paper
title: "TumorFusions: an integrative resource for cancer-associated transcript fusions"
image: /publications/data/2017_NAR_TumorFusions.png
authors: Hu X, Wang Q, Tang M, Barthel F, Amin S, Yoshihara K, Lang FM, Martinez-Ledesma E, Lee SH, Zheng S, Verhaak RGW
year: 2018
ref: Hu et al. 2018. Nucleic Acids Res.
journal: "Nucleic Acids Res. 2018 Jan 4;46(D1):D1144-D1149"
pdf: /publications/data/2017_NAR_TumorFusions.pdf
doi: 10.1093/nar/gkx1018
pmid: 29099951
pmcid: PMC5753333
---

# Abstract

Gene fusion represents a class of molecular aberrations in cancer and has been exploited for therapeutic purposes. In this paper we describe TumorFusions, a data portal that catalogues 20 731 gene fusions detected in 9966 well characterized cancer samples and 648 normal specimens from The Cancer Genome Atlas (TCGA). The portal spans 33 cancer types in TCGA. Fusion transcripts were identified via a uniform pipeline, including filtering against a list of 3838 transcript fusions detected in a panel of 648 non-neoplastic samples. Fusions were mapped to somatic DNA rearrangements identified using whole genome sequencing data from 561 cancer samples as a means of validation. We observed that 65% of transcript fusions were associated with a chromosomal alteration, which is annotated in the portal. Other features of the portal include links to SNP array-based copy number levels and mutational patterns, exon and transcript level expressions of the partner genes, and a network-based centrality score for prioritizing functional fusions. Our portal aims to be a broadly applicable and user friendly resource for cancer gene annotation and is publicly available at http://www.tumorfusions.org.